.github/workflows/codeql.yml

```diff
@@ -0,0 +1,68 @@
+name: "CodeQL code scanning"
+
+on:
+  push:
+    branches: [ "master", "beta", "rc" ]
+  pull_request:
+    branches: [ "master", "beta", "rc" ]
+  schedule:
+    - cron: '35 22 * * 3'
```

Comment:
Will this run on all branches?

Reply:
no - only master: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule